### PR TITLE
add some examples

### PR DESCRIPTION
**Pull request by n-h-diaz**

### CODE REVIEW

File: static/js/tools/map/info.tsx [hunk 1]
```diff
@@ -40,6 +40,61 @@ export function Info(): JSX.Element {
...
+          <a
+            href={
+              "#%26sv%3DDifferenceRelativeToBaseDate2006_Max_Temperature_RCP45%26pc%3D0%26denom%3DCount_Person%26pd%3Dcountry%2FIND%26pn%3DIndia%26pt%3DCountry%26ept%3DAdministrativeArea2"
+            }
+          >
+            India
+          </a>
+        </li>
+        <li>
+          <b>Annual Amount of Non Biogenic Emission Source, Carbon Dioxide</b>{" "}
```
Comment (beets) [inline static/js/tools/map/info.tsx:62]:
```suggestion
          <b>Annual Amount of Non-Biogenic Emission Source, Carbon Dioxide</b>{" "}
```

File: static/js/tools/scatter/info.tsx [hunk 2]
```diff
@@ -40,6 +40,61 @@ function Info(): JSX.Element {
...
+          <a
+            href={
+              "#%26svx%3DMean_SolarInsolation%26svy%3DCount_Person_BelowPovertyLevelInThePast12Months%26pcy%3D1%26epd%3DgeoId%2F06%26epn%3DCalifornia%26epts%3DState%26ept%3DCounty%26qd%3D0%26ld%3D0%26dd%3D0%26ct%3D1"
+            }
+          >
+            California
+          </a>
+        </li>
+        <li>
+          <b>Literate Population Per Capita vs Count Of School Per Capita</b>{" "}
```
Comment (beets) [inline static/js/tools/scatter/info.tsx:88]:
```suggestion
          <b>Literate Population Per Capita vs Count Of Schools Per Capita</b>{" "}
```

**Review by beets — APPROVED** [review]
thanks for htese!

File: static/js/tools/scatter/info.tsx (hunk 2 above)
Comment (n-h-diaz) [reply static/js/tools/scatter/info.tsx]: done

File: static/js/tools/map/info.tsx (hunk 1 above)
Comment (n-h-diaz) [reply static/js/tools/map/info.tsx]: done

File: static/js/tools/scatter/info.tsx (hunk 2 above)
Comment (n-h-diaz) [reply static/js/tools/scatter/info.tsx]: thanks for the review!

**Pull request merged**